build.gradle

```diff
@@ -41,12 +42,33 @@ ext {
     dockerBuildAllImagesStageName = 'dockerBuildAllImages'
     dockerLoginDockerHubStageName = 'dockerLoginDockerHub'
     dockerPublishAllImagesStageName = 'dockerPublishAllImages'
+
+    helmArtifactoryRepos = []
+    helmPrePackagedRepoNames = []
+
+    envVarsNotExistMsg = 'Skipping as dependent environment variable are not set'
 }
 
 mainClassName = 'com.synopsys.integration.alert.Application'
 version = '7.0.0-SNAPSHOT'
 
 ext.isSnapshot = project.version.endsWith('-SNAPSHOT')
+ext.isSIGQA = project.version.contains('-SIGQA')
+ext.isRelease = (!project.ext.isSnapshot && !project.ext.isSIGQA)
+
+if (project.ext.isSIGQA && System.env.HELM_SIGQA_REPOS != null) {
+    helmArtifactoryRepos = System.env.HELM_SIGQA_REPOS.split(',').collect { it.trim() as String }
+} else if (project.ext.isSnapshot && System.env.HELM_SNAPSHOT_REPOS != null) {
+    helmArtifactoryRepos = System.env.HELM_SNAPSHOT_REPOS.split(',').collect { it.trim() as String }
+} else if (project.ext.isRelease && System.env.HELM_RELEASE_REPOS != null) {
+    helmArtifactoryRepos = System.env.HELM_RELEASE_REPOS.split(',').collect { it.trim() as String }
+} else {
+    logger.lifecycle("Helm Artifactory repositores not configured")
+}
```

Comment:
We have 4 different "types" for version:

* 7.0.0-SNAPSHOT - This is a snapshot build
* 7.0.0-SIGQA1-SNAPSHOT - This is a SIGQA build
* 7.0.0-SIGQA1 - This is a SIGQA build
* 7.0.0 - This is a release build

Because of the SIGQA1-SNAPSHOT naming convention, the if statement needs to check first if it's a SIGQA build